### PR TITLE
add ISSUE_TEMPLATE file. Fixes #2735

### LINKED ISSUE #2735: Issue Template
With only a comment explaining what info there should be, not forcing the user or scaring him off.

difficulty/low

### PR DESCRIPTION
**Pull request by peey**
This is how it would appear when someone is on the new-issue page:

Please comment if you feel there could be improvements. Otherwise consider for merging.

### CODE REVIEW

File: ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!-- Hello! If you're filing a bug, please include every step so as to help us reproduce it on our machines. If you're unsure about how to file an issue, use the issue template. If you need any help regarding usage of coala, check out the documentation or hit us up on the chat-room. You can ignore or delete this text, it is commented and won't appear when the issue is submitted or previewed.
+
+Chat-room: https://coala.io/chat
+Issue Template: https://github.com/coala/coala/blob/master/CONTRIBUTING.rst#filing-issues
+Documentation: https://docs.coala.io/en/latest/
```
Comment (sils) [inline ISSUE_TEMPLATE]: remove `en/latest/`

Comment (peey) [inline ISSUE_TEMPLATE]: done!

File: ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<!-- Hello! If you're filing a bug, please include every step so as to help us reproduce it on our machines. If you're unsure about how to file an issue, use the issue template. If you need any help regarding usage of coala, check out the documentation or hit us up on the chat-room. You can ignore or delete this text, it is commented and won't appear when the issue is submitted or previewed.
```
Comment (adtac) [inline ISSUE_TEMPLATE]: Could you please take a look at http://coala.io/commit and ensure all points are followed? Thanks :)

Comment (adtac) [inline ISSUE_TEMPLATE]: Marking as WIP till then.

Comment (peey) [inline ISSUE_TEMPLATE]: I have done it already :)  If you find any specific point which I missed then please let me know

Comment (peey) [inline ISSUE_TEMPLATE]: Any updates on this?

Comment (sils) [inline ISSUE_TEMPLATE]: add -> Add, otherwise it's good IMO

File: ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+Chat-room: https://coala.io/chat
```
Comment (sils) [inline ISSUE_TEMPLATE]: IMO simply do chat, without the room (also in the text above)

File: ISSUE_TEMPLATE [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+Issue Template: https://github.com/coala/coala/blob/master/CONTRIBUTING.rst#filing-issues
+Documentation: https://docs.coala.io
+-->
```
Comment (sils) [inline ISSUE_TEMPLATE]: missing newline before EOF

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/2853/commits/985b9cf759f95f23caf65c455e5ae61759e42dcd

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
Hey, this is your first contribution, right? Congrats and welcome to the coalaians! :)

You can help us a lot by filling out <http://coala.io/newform>!

Rultor will now automatically release your changes as a prerelease so users can experience the goodness of your changes right now.

See also https://github.com/coala-analyzer/coala/wiki/coala-Steam-Offer btw.

We would really appreciate it if you could help us with

- some code reviews for other newcomers
- fixing at least a difficulty/low issue

We're all volunteers and we're currently struggling to keep this up. Helping all those newcomers is a lot of work and we do need your help so we can continue this for others!

**Pull request merged**